python-sdk/tests_integration/sql/operators/test_load_file.py

```diff
@@ -515,7 +505,7 @@ def test_load_file_chunks(sample_dag, database_table_fixture):
         ),
     ],
     indirect=["database_table_fixture"],
-    ids=["Bigquery", "Redshift"],
+    ids=["Redshift"],
```

Comment:
Why did we remove BQ?

Reply:
+1

Reply:
@tatiana this one test took 5 minutes to complete and as far as I can tell it's not really testing anything that's not already tested (note that code coverage has not changes at all)